scripts/build.js

```diff
@@ -356,57 +404,49 @@ async function buildTypes () {
 		noEmitOnError: false,
 	};
 
-	const program = ts.createProgram(parsedConfig.fileNames, compilerOptions);
+	// Get all .js files from the src directory
+	const files = await findFiles(SRC_DIR);
+
+	const program = ts.createProgram(files, compilerOptions);
```

Comment:
And the last part of it.

It's straightforward, but you might find flaws we need to fix.